core/src/test/java/org/apache/calcite/test/RexProgramTest.java

```diff
@@ -228,6 +228,18 @@ private RexNode isNotNull(RexNode node) {
     return rexBuilder.makeCall(SqlStdOperatorTable.IS_NOT_NULL, node);
   }
 
+  private RexNode isFalse(RexNode node) {
+    return rexBuilder.makeCall(SqlStdOperatorTable.IS_FALSE, node);
+  }
+
+  private RexNode isTrue(RexNode node) {
+    return rexBuilder.makeCall(SqlStdOperatorTable.IS_TRUE, node);
+  }
+
+  private RexNode isNotTrue(RexNode node) {
```

Comment:
Could you please co-locate `is*True/False` methods? It looks like `isNotFalse` is declared elsewhere

Reply:
Thanks! I have added the isNotFalse method after isNotTrue.